js/src/lib/components/InferenceWidget/shared/WidgetExample.ts

```diff
@@ -0,0 +1,94 @@
+import type { TableData } from "./types";
+
+//#region outputs
+export type WidgetExampleOutputLabels = Array<{ label: string; score: number; }>;
+export interface WidgetExampleOutputAnswerScore {
+	answer: string;
+	score:  number;
+}
+export interface WidgetExampleOutputText {
+	text: string;
+}
+export interface WidgetExampleOutputUrl {
+	url: string;
+}
+//#endregion
+
+export interface WidgetExampleBase {
+	example_title?: string;
+	group?:         string;
+}
```

Comment:
```suggestion
export interface WidgetExampleBase<T> {
	example_title?: string;
	group?:         string;
	output?:        T;
}
```

since `output` attribute could be present in all the widget example types, maybe we can use generic here?

Reply:
cc @SBrandeis WDYT?

Reply:
i tried but it didn't really simplify things.

That being said, i'm not great at types, so maybe you can try making it work better in a later PR?

Reply:
yeah that works! Therefore, resolving this issue for now